packages/vaadin-combo-box/test/dynamic-size.test.js

```diff
@@ -31,8 +32,9 @@ describe('dynamic size change', () => {
       comboBox.dataProvider = dataProvider;
     });
 
-    it('should not have item placeholders after size gets reduced', () => {
+    it('should not have item placeholders after size gets reduced', async () => {
       comboBox.opened = true;
+      await nextRender(comboBox.$.overlay);
```

Comment:
There is no `comboBox.$.overlay` anymore. Also, in fact the reference passed to `nextRender` doesn't really matter: Polymer uses it as `this` reference to call [`.apply()`](https://github.com/Polymer/polymer/blob/10220c9af132149c31cd0539273aad1be251fcb4/lib/utils/render-status.js#L53) but in `nextRender` we use arrow function.